modules/ascat/meta.yml

```diff
@@ -0,0 +1,68 @@
+name: ascat
+description: copy number profiles of tumour cells.
+keywords:
+  - sort
+tools:
+  - ascat:
+      description: ASCAT is a method to derive copy number profiles of tumour cells, accounting for normal cell admixture and tumour aneuploidy. ASCAT infers tumour purity (the fraction of tumour cells) and ploidy (the amount of DNA per tumour cell), expressed as multiples of haploid genomes from SNP array or massively parallel sequencing data, and calculates whole-genome allele-specific copy number profiles (the number of copies of both parental alleles for all SNP loci across the genome).
+      homepage: None
+      documentation: None
+      tool_dev_url: https://github.com/Crick-CancerGenomics/ascat
+      doi: ""
+      licence: ['GPL v3']
+
+input:
```

Comment:
I would propose to add the args template here with markdown, so the user knows which fields they need to fill and which are optional/spelling etc.

Reply:
sorry I see this now, but as far as I know ASCAT has a publication, can you link the doi?

Reply:
Sounds like someone is planning documentation hackaton soon 😄 
I'll create an issue for it, both so we have something for the hackaton, but also so I can see 'how it's done', because I actually I have no idea where and how to document arguments currently.

Reply:
@mashehu advised me here to write a code block with markdown https://github.com/nf-core/modules/blob/79a9d5e1eae6cc6f3b2a0a0d02e61ae4b5ef3748/modules/controlfreec/meta.yml#L19 I don't think there is a standardized way yet to document `args` (also seems to be a rare problem, as you can normally just provide a string)

Reply:
Ok, I put it in like in controlfreec for now. I bet it's something we'll discuss more at the hackaton.